pkgs/development/r-modules/generate-r-packages.R

```diff
@@ -72,8 +79,8 @@ pkgs <- as.data.table(available.packages(mirrorUrl, filters=c("R_version", "OS_t
 pkgs <- pkgs[order(Package)]
 
 write(paste("updating", mirrorType, "packages"), stderr())
-pkgs$sha256 <- parApply(cl, pkgs, 1, function(p) nixPrefetch(p[1], p[2]))
```

Comment:
These changes are unrelated to the commit message. What issue/bug is being resolved here? Can be split into a separate commit addressing the bug.

Reply:
I'll have to check this one, it doesn't look right to me anymore.

Reply:
Reverted, only seemed to matter when internet connectivity failed during package generation, but then other issues occured, where SHA fields came out empty.